Lab5/GABDemo/Startup.cs

```diff
@@ -31,9 +31,17 @@ public void ConfigureServices(IServiceCollection services)
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
-            services.Configure<KeysOptions>(Configuration.GetSection("Keys"))
-                    .PostConfigure<KeysOptions>(options =>
+            services.Configure<KeysOptions>(options =>
             {
+                options.ComputerVision = new ComputerVisionOptions
+                {
+                    ApiEndPoint = Configuration["Keys:ComputerVision:ApiEndPoint"],
+                    ApiKey = Configuration["Keys:ComputerVision:ApiKey"]
+                };
+                options.Storage = new StorageAccountOptions { ConnectionString = Configuration.GetConnectionString("ApplicationStorage") };
```

Comment:
What does *Keys* mean in `KeysOptions`? It is not clear.

If both `ComputerVisionOptions` and `StorageAccountOptions` are related to `KeysOptions`, why extract the `ApplicationStorage` connection string out of the `Keys` object (in *appsettings.json*)?

If the connection string does not belong to `Keys` (in *appsettings.json*), it should not belong to `KeysOptions` either, which should lead to the deletion of that object and to load both `ComputerVisionOptions` and `StorageAccountOptions` individually (both options are unrelated).